client/src/app/shared/video/abstract-video-list.html

```diff
@@ -1,8 +1,18 @@
 <div [ngClass]="{ 'margin-content': marginContent }">
   <div class="videos-header">
     <div *ngIf="titlePage" class="title-page title-page-single">
-      {{ titlePage }}
+      <ng-template i18n #tipContent>
+        The <b>Trending algorithm</b> sums the views made in the last {{ trendingDays }} days for the videos of the related instances, and sort them.
```

Comment:
I think we should remove the `<b>` tag, so we can put this text in a `titleTooltip` variable in `abstract-video-list.ts` (the children choose to fill it or not). This way, you would not have this ngIf in the template:

```
<div placement="bottom" [ngbTooltip]="titleTooltip" container="body">
  {{ titlePage }}
</div>
```

In the trending videos component, `titlePage` should be `titlePage = this.i18n('{{titlePage}} for the last {{trendingDays}} days', ...)`

Reply:
Thanks for this much more elegant solution !